src/asm/rgbasm.5

```diff
@@ -1010,15 +1019,22 @@ Note that colons
 following the name are not allowed.
 .It Ic EQUS
 .Ic EQUS
-is used to define string symbols.
-Wherever the assembler meets a string symbol its name is replaced with its value.
-If you are familiar with C you can think of it as similar to
+is used to define string equate symbols.
+Wherever the assembler reads a string equate, the symbol is expanded as if its contents had been read instead.
```

Comment:
I think we should explicit that it's as soon as the EQUS' *name* is encountered, that the expansion takes place.

Reply:
How about "the symbol's name is expanded as if its contents had been there instead." ?

Reply:
Perhaps "replaced" instead of "expanded"? Otherwise, yes.

Reply:
Okay.